Composer/packages/client/src/recoilModel/DispatcherWrapper.tsx

```diff
@@ -55,7 +55,7 @@ const wrapDispatcher = (dispatchers, forceUpdate) => {
   return Object.keys(dispatchers).reduce((boundDispatchers, dispatcherName) => {
     const dispatcher = async (...args) => {
       forceUpdate([]); //gurarantee the snapshot get the latset state
-      await dispatchers[dispatcherName](...args);
+      return await dispatchers[dispatcherName](...args);
```

Comment:
We should not be returning values from dispatchers. They should only be used to update state, then components subscribe to state updates to re-render.